e2e/e2e_partition_test.go

```diff
@@ -6,9 +6,100 @@ import (
 	"testing"
 	"time"
 
+	"github.com/0xPolygon/pbft-consensus"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestE2E_Partition_Liveness(t *testing.T) {
```

Comment:
Please add a command here with a reference to the paper so that we know exactly what issue is this covering.